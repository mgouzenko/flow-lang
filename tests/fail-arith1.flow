int main() {
  int a = 1;
  string b = "hey";
  int c = a + b;
  return 0;
}