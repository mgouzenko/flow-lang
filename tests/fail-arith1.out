Fatal error: exception Invalid_argument("operator + not compatible with int and string")
